wallet/bip32.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 
 	smbip32 "github.com/spacemeshos/smkeys/bip32"
+	ledger "github.com/spacemeshos/smkeys/remote-wallet"
```

Comment:
you can add the other module by explicitly referencing the commit; are you using `go work` locally?

```
go get github.com/spacemeshos/smkeys@15e5e907
```